core/src/main/java/io/grpc/ServerCall.java

```diff
@@ -167,4 +167,12 @@ public boolean isReady() {
    * <p>This method may safely be called concurrently from multiple threads.
    */
   public abstract boolean isCancelled();
+
+  /**
+   * Enables per-message compression, if an encoding type has been negotiated.  If no message
```

Comment:
If unset, we as library owners I believe make the decision about the default.  In order to not suddenly add a CPU spike to users I left it as is.  It is true that they would have to set this per call, but as this is in experimental mode I believe this is okay.